src/test/rustdoc-gui/jump-to-def-background.goml

```diff
@@ -0,0 +1,23 @@
+// We check the background color on the jump to definition links in the source code page.
+goto: file://|DOC_PATH|/src/test_docs/lib.rs.html
+
+// Set the theme to dark.
+local-storage: {"rustdoc-theme": "dark", "rustdoc-preferred-dark-theme": "dark", "rustdoc-use-system-theme": "false"}
+// We reload the page so the local storage settings are being used.
+reload:
+
+assert-css: ("body.source .example-wrap pre.rust a", {"background-color": "rgb(51, 51, 51)"}, ALL)
+
+// Set the theme to ayu.
+local-storage: {"rustdoc-theme": "ayu", "rustdoc-preferred-dark-theme": "ayu", "rustdoc-use-system-theme": "false"}
```

Comment:
Not specific to this PR, but I'm curious: is it necessary to set the preferred dark theme given that it says to ignore the system theme and manually specifies the current theme?

Reply:
Not sure it is, but just in case... :)

Reply:
Ok, that sounds fine 👍